### ISSUE TITLE
Wrong default language - WCAG 3.1.2 failure

### ISSUE THREAD

**Issue by HendrikWismans** [elmarquis/Leaflet.GestureHandling#86]
The libary is multilangue but use the wrong (system) default. Is it possible to use the language (if available) defined in the html attribute? It's not allowed to use several languages in one html page without lang attribute. This is a WCAG 3.1.2 failure.
